src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -247,6 +247,8 @@
             <xsd:element name="default-memcached-provider" type="xsd:string" minOccurs="0" maxOccurs="1" />
             <xsd:element name="pool" type="cache_pool" minOccurs="0" maxOccurs="unbounded" />
         </xsd:sequence>
+        <xsd:attribute name="version" type="xsd:string"/>
+        <xsd:attribute name="prefix-seed" type="xsd:string"/>
```

Comment:
adding this missing attribute in the XSD is a bug fix though. Can you submit a PR to the 3.4 branch with this change only ?

Reply:
Changed